# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../dovecot-pigeonhole/dovecot-2.2-pigeonhole-0.4.0-doc-hotfix.patch
# Copyright (C) 2013 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---


# HG changeset patch
# User Stephan Bosch <stephan@rename-it.nl>
# Date 1368110381 -7200
# Node ID a32b12ab5ea6c8fcc3f4971ddceb8e4454ee84e5
# Parent  1e0a6a4f4634fe3559cef631dae46f52ec2bdf47
Forgot to update Dovecot version in README and man pages.

diff -r 1e0a6a4f4634 -r a32b12ab5ea6 README
--- a/README	Thu May 09 15:41:42 2013 +0200
+++ b/README	Thu May 09 16:39:41 2013 +0200
@@ -1,4 +1,4 @@
-Pigeonhole for Dovecot v2.1
+Pigeonhole for Dovecot v2.2
 
 Introduction
 ============
@@ -100,8 +100,7 @@
 
     copy (RFC 3894): fully supported.
     body (RFC 5173): almost fully supported, but the text body-transform
-        implementation is simple and some issues make it still not completely
-        RFC compliant.
+        implementation is simple.
     environment (RFC 5183): fully supported (v0.4.0+).
     variables (RFC 5229): fully supported.
     vacation (RFC 5230): fully supported.
diff -r 1e0a6a4f4634 -r a32b12ab5ea6 doc/man/pigeonhole.7.in
--- a/doc/man/pigeonhole.7.in	Thu May 09 15:41:42 2013 +0200
+++ b/doc/man/pigeonhole.7.in	Thu May 09 16:39:41 2013 +0200
@@ -1,5 +1,5 @@
-.\" Copyright (c) 2010-2012 Pigeonhole authors, see the included COPYING file
-.TH "PIGEONHOLE" 7 "2012-01-07" "Pigeonhole for Dovecot v2.1" "Pigeonhole"
+.\" Copyright (c) 2010-2013 Pigeonhole authors, see the included COPYING file
+.TH "PIGEONHOLE" 7 "2013-05-09" "Pigeonhole for Dovecot v2.2" "Pigeonhole"
 .\"------------------------------------------------------------------------
 .SH NAME
 pigeonhole \- Overview of the Pigeonhole project\(aqs Sieve support for the
diff -r 1e0a6a4f4634 -r a32b12ab5ea6 doc/man/sieve-dump.1.in
--- a/doc/man/sieve-dump.1.in	Thu May 09 15:41:42 2013 +0200
+++ b/doc/man/sieve-dump.1.in	Thu May 09 16:39:41 2013 +0200
@@ -1,5 +1,5 @@
-.\" Copyright (c) 2010-2012 Pigeonhole authors, see the included COPYING file
-.TH "SIEVE\-DUMP" 1 "2012-01-07" "Pigeonhole for Dovecot v2.1" "Pigeonhole"
+.\" Copyright (c) 2010-2013 Pigeonhole authors, see the included COPYING file
+.TH "SIEVE\-DUMP" 1 "2013-05-09" "Pigeonhole for Dovecot v2.2" "Pigeonhole"
 .\"------------------------------------------------------------------------
 .SH NAME
 sieve\-dump \- Pigeonhole\(aqs Sieve script binary dump tool
diff -r 1e0a6a4f4634 -r a32b12ab5ea6 doc/man/sieve-filter.1.in
--- a/doc/man/sieve-filter.1.in	Thu May 09 15:41:42 2013 +0200
+++ b/doc/man/sieve-filter.1.in	Thu May 09 16:39:41 2013 +0200
@@ -1,5 +1,5 @@
-.\" Copyright (c) 2010-2012 Pigeonhole authors, see the included COPYING file
-.TH "SIEVE\-FILTER" 1 "2012-01-07" "Pigeonhole for Dovecot v2.1" "Pigeonhole"
+.\" Copyright (c) 2010-2013 Pigeonhole authors, see the included COPYING file
+.TH "SIEVE\-FILTER" 1 "2013-05-09" "Pigeonhole for Dovecot v2.2" "Pigeonhole"
 .SH NAME
 sieve\-filter \- Pigeonhole\(aqs Sieve mailbox filter tool
 
diff -r 1e0a6a4f4634 -r a32b12ab5ea6 doc/man/sieve-test.1.in
--- a/doc/man/sieve-test.1.in	Thu May 09 15:41:42 2013 +0200
+++ b/doc/man/sieve-test.1.in	Thu May 09 16:39:41 2013 +0200
@@ -1,5 +1,5 @@
-.\" Copyright (c) 2010-2012 Pigeonhole authors, see the included COPYING file
-.TH "SIEVE\-TEST" 1 "2012-01-07" "Pigeonhole for Dovecot v2.1" "Pigeonhole"
+.\" Copyright (c) 2010-2013 Pigeonhole authors, see the included COPYING file
+.TH "SIEVE\-TEST" 1 "2013-05-09" "Pigeonhole for Dovecot v2.2" "Pigeonhole"
 .SH NAME
 sieve\-test \- Pigeonhole\(aqs Sieve script tester
 .\"------------------------------------------------------------------------
diff -r 1e0a6a4f4634 -r a32b12ab5ea6 doc/man/sievec.1.in
--- a/doc/man/sievec.1.in	Thu May 09 15:41:42 2013 +0200
+++ b/doc/man/sievec.1.in	Thu May 09 16:39:41 2013 +0200
@@ -1,5 +1,5 @@
-.\" Copyright (c) 2010-2012 Pigeonhole authors, see the included COPYING file
-.TH "SIEVEC" 1 "2012-01-07" "Pigeonhole for Dovecot v2.1" "Pigeonhole"
+.\" Copyright (c) 2010-2013 Pigeonhole authors, see the included COPYING file
+.TH "SIEVEC" 1 "2013-05-09" "Pigeonhole for Dovecot v2.2" "Pigeonhole"
 .\"------------------------------------------------------------------------
 .SH NAME
 sievec \- Pigeonhole\(aqs Sieve script compiler

